main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Enter shares bought." << std::endl;
+    int shares_bought = 0;
+    std::cin >> shares_bought;
+    std::cout << "Enter cost."  <<std::endl;
+    double purchase_cost = 0.0;
+    std::cin >> purchase_cost;
+    double COMMISSION_RATE = .02;
```

Comment:
Needs the `const` keyword to be a constand and should be hoisted to the top of the scope